### ISSUE TITLE
Warlock Quest " An Imps Request" isn't available until phase 3. 

### ISSUE THREAD

**Issue by Basih149** [AeroScripts/QuestieDev#1088]
## Description
The quest isn't in the game and according to a comment on the WoWhead page for the quest it isn't available until phase 3. It serves as a catch-up later.

## Wowhead link
https://classic.wowhead.com/quest=8419

## Screenshots
Don't have a screenshot as I removed the quest manually but it is where it was supposed to be after phase 3 is introduced. 

## Questie version
Questie Version v4.1.1 Beta

**Comment by BreakBB** [comment]
Duplicate of #1044
Fixed with 21d045c6faf6deff995d4a3c68e190ab705f3509

And according to [Wowhead itself](https://www.wowhead.com/news=295107/wow-classic-sunken-temple-class-quests-not-available-in-phase-1) the Sunken Temple class quests will most likely be available if Phase 4
